packages/web/src/components/DevFatalErrorPage.tsx

```diff
@@ -7,7 +7,7 @@ import { useState } from 'react'
 import StackTracey from 'stacktracey'
 
 // RWJS_SRC_ROOT is defined and defaulted in webpack to the base path
-const srcRoot = process.env.RWJS_SRC_ROOT || ''
+const srcRoot = RWJS_DEBUG_ENV?.RWJS_SRC_ROOT || ''
```

Comment:
I don't understand the nuance yet so just going to ask: why no `globalThis`?

Reply:
When you use `globalThis` - it fails to find the variable in either Webpack or Vite.

Directly accessing the variable like this is the only way I could make sure it works for both!